services/harmony/app/util/parameter-parsers.ts

```diff
@@ -79,10 +79,25 @@ export function handleCrs(
  */
 export function handleScaleExtent(
```

Comment:
TypeScript didn't like just using a union of types for the `query` unless I overloaded the function.

Reply:
I was able to just use the last definition without issue - maybe a VS code problem? Can you check what I pushed up to the branch to make sure you don't have any issues?

Reply:
I pulled the new commit and my editor (vim :wink:)  seemed to like it, as did running the tests/type-checking. I think at the same time I implemented the overloads, I also cut over from `instanceof` to `typeof`, so that's probably really what quashed my type errors (and seems pretty obvious now I write it down 🙄).